src/appendices/contributors.md

```diff
@@ -55,13 +56,14 @@ If you contributed to the BIDS ecosystem and your name is not listed, please add
 | Alexander Jones                                      | 💻🐛                                   |
 | Alexander L. Cohen                                   | 🐛💻📖💬                               |
 | Alexander von Lautz                                  | 📖                                     |
-| Alexandre D'Astous                                   | 📖                                     |
```

Comment:
@po09i I'm assuming you didn't remove yourself.

@Remi-Gau Do you know what happened here?

Reply:
Hypothesis: Alexandre was added only to the text file here in https://github.com/bids-standard/bids-specification/pull/1690/files but never to any of the "metadata" files for contributors. Thus whenever regenerated -- disappeared?  So needs to gain a record in .tributors and .all-contributorsrc?

Reply:
yeah I am pretty sure this it what happened: will fix